### DISCUSSION TITLE
NFC Navigo compatibility

### DISCUSSION THREAD

**Discussion by bens0607** [nabaztag2018/pynab#437] [Ideas]
I read that the navigo pass could be compatible with the NFC 2022 card. 

Is the functionality planned?

Here is the UID code indicated on the web interface after reading by the rabbit: 26:55:44:30 (iso14443b).

Thank you very much for all the happiness brought by the new kit and the code that goes with it.

Have a nice evening.

**Comment by f-laurens** [comment]
See: https://github.com/nabaztag2018/pynab/issues/408.
